src/sentry/api/base.py

```diff
@@ -184,7 +184,7 @@ def build_cursor_link(self, request: Request, name: str, cursor: Cursor):
             mutable_query_dict.pop("cursor")
             querystring = mutable_query_dict.urlencode()
 
-        base_url = request.build_absolute_uri(urlquote(request.path))
+        base_url = absolute_uri(urlquote(request.path))
```

Comment:
One thing we lose by using `absolute_uri` is that we force the hostname to be the control silo hostname. But probably not a big deal.

e.g. forcing any `orgslug.sentry.io` to become `sentry.io`.

Reply:
That's actually a good point, and a potential problem.  I think we unfortunately need more test cases and more investment here, I'll take a look shortly.

Reply:
@corps there's a `url_prefix` parameter that we can use. We've been using it for customer domains in various places: https://github.com/getsentry/sentry/blob/81dd132d0655e0704d0a69c6f5006b5179d235a5/src/sentry/web/frontend/base.py#L218-L220

So we can add it only whenever `using_customer_domain = request and is_using_customer_domain(request)`